addons/google_calendar/models/res_users.py

```diff
@@ -186,3 +186,17 @@ def check_synchronization_status(self):
                 sync_status = 'sync_stopped'
         res['google_calendar'] = sync_status
         return res
+
+    def _has_any_active_synchronization(self):
+        """
+        Check if synchronization is active for Google Calendar.
+        This function retrieves the synchronization status from the user's environment
+        and checks if the Google Calendar synchronization is active.
+
+        :return: Action to delete the event
+        """
+        sync_status = self.check_synchronization_status()
+        res = super()._has_any_active_synchronization()
+        if sync_status.get('google_calendar') == 'sync_active':
+            res |= True
```

Comment:
Why not just returning True? res |= True is a strange operator for a bool :thinking: 
If res was False, you want to return False. If it was already True, you still want to return True.